src/main/tls.c

```diff
@@ -3091,6 +3091,13 @@ SSL_CTX *tls_init_ctx(fr_tls_server_conf_t *conf, int client)
 	ctx_options |= SSL_OP_NO_SSLv2;
 	ctx_options |= SSL_OP_NO_SSLv3;
 
+	/*
+	 *	EAP (hopefully) does not have middlebox deployments
+	 */
+#ifdef SSL_OP_ENABLE_MIDDLEBOX_COMPAT
```

Comment:
I'd add a comment about "If set then dummy Change Cipher Spec (CCS) messages are sent in TLSv1.3. This has the effect of making TLSv1.3 look more like TLSv1.2 so that middleboxes that do not understand TLSv1.3 will not drop the connection.  This isn't needed for EAP-TLS, so we disable it"

Reply:
Resolved by https://github.com/FreeRADIUS/freeradius-server/pull/3516/commits/becf52a42962070854e771019b08b97197b14588